### PR TITLE
add documentation for get model group API

### PR DESCRIPTION
**Pull request by rbhavna**
### Description
add documentation for get model group API

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+layout: default
+title: Get model group
+parent: Model group APIs
+grand_parent: ML Commons APIs
+nav_order: 30
+---
+
+# Get a model group
+
+This API is available from 2.13 release
```
Comment (dhrubo-os) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:11]: Is this correct?

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{: .important}
+
+If model access control is enabled on your cluster, only the owner or users with matching backend roles can get the model group. Any users can get any public model group.
+
+If model access control is disabled on your cluster, users with the `get model group API` permission can get any model group.
+
+Admin users can get any model group.
```
Comment (ylwu-amzn) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:18]: Does hidden model has model group ?

Comment (rbhavna) [reply _ml-commons-plugin/api/model-group-apis/get-model-group.md:18]: Yes but model group is not hidden. We don't have hidden model group.

**Review by dhrubo-os — APPROVED** [review]

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md (hunk 2 above)
Comment (ylwu-amzn) [reply _ml-commons-plugin/api/model-group-apis/get-model-group.md:18]: Got it, so admin can still get hidden model group

**Review by ylwu-amzn — APPROVED** [review]

**Comment by hdhalter** [comment]
Should this be backported to 2.12?

**Comment by dhrubo-os** [comment]
> Should this be backported to 2.12?

Yes

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md (hunk 2 above)
Comment (rbhavna) [reply _ml-commons-plugin/api/model-group-apis/get-model-group.md:18]: yes

**Comment by rbhavna** [comment]
> Should this be backported to 2.12?

yes that would be great

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md (hunk 1 above)
Comment (rbhavna) [reply _ml-commons-plugin/api/model-group-apis/get-model-group.md]: changed it to 2.12

**Review by Naarcha-AWS — APPROVED** [review]

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+grand_parent: ML Commons APIs
+nav_order: 30
+---
+
+# Get a model group
+
+Introduced 2.12
+{: .label .label-purple }
+
+The Get model group API returns information about a model group based on that group's ID.
```
Comment (natebower) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:14]: Get Model Group API?

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+# Get a model group
+
+Introduced 2.12
+{: .label .label-purple }
+
+The Get model group API returns information about a model group based on that group's ID. 
+
+If model access control is enabled on your cluster, only the owner or users with matching backend roles can get private model groups. Any users can get any public model group.
```
Comment (natebower) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:16]:
```suggestion
If model access control is enabled on your cluster, only the owner or users with matching backend roles can get private model groups. Any user can get any public model group.
```

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+The following example request gets a model group based on that group's ID:
+
+```json
+GET /_plugins/_ml/model_groups/<model_group_id>
+```
+{% include copy-curl.html %}
+
+### Example response
+
+The following response returns the model group information, including the model group's name, version, description, access level, and creation information.
```
Comment (natebower) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:39]:
```suggestion
The following response returns the model group information, including the model group's name, version, description, access level, and creation information:
```

**Review by natebower — COMMENTED** [review]
@rbhavna @Naarcha-AWS Please see my comments and changes and let me know if you have any questions. Thanks!

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Get a model group
```
Comment (Naarcha-AWS) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:3]:
```suggestion
title: Get model group
```

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+parent: Model group APIs
+grand_parent: ML Commons APIs
+nav_order: 30
+---
+
+# Get a model group
```
Comment (Naarcha-AWS) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:9]:
```suggestion
# Get model group
```

File: _ml-commons-plugin/api/model-group-apis/get-model-group.md (hunk 3 above)
Comment (Naarcha-AWS) [inline _ml-commons-plugin/api/model-group-apis/get-model-group.md:14]:
```suggestion
The Get Model Group API returns information about a model group based on that group's ID. 
```

**Pull request merged**